pkg/ccl/backupccl/restore_online_test.go

```diff
@@ -57,6 +60,12 @@ func TestOnlineRestoreBasic(t *testing.T) {
 
 	sqlDB.Exec(t, fmt.Sprintf("BACKUP INTO '%s'", externalStorage))
 
+	if rng.Float64() < 0.5 {
```

Comment:
I'm a little wary of the randomness just making it flake; any reason not to just have two tests -- one inc one no inc?

Reply:
each test takes 5s 😱

Reply:
:(

I'm extremely sympathetic to slow tests being something to avoid.

That said, I still think correctness of incremental is worth having a test that reliably and deterministically runs it. If the test is slow we could check the backups in instead of needing to load bank and back it up every time.